develop/tutorials/articles/140-assets/rendering-an-asset.markdown

```diff
@@ -371,15 +370,18 @@ asset renderer is put together to satisfy JSP template development requirements.
             }
         }
 
-    Blogs assets implement the `abstract.jsp` and `full_content.jsp` templates.
+    Blogs assets provide custom `abstract.jsp` and `full_content.jsp` templates.
     This means that a blogs asset can render a blog's abstract description or
     the blog's full content in the Asset Publisher. Those templates are located
     in the `com.liferay.blogs.web` module's
     `src/main/resources/META-INF/resources/blogs/asset` folder. You could create
-    a similar folder for your JSP templates used for this method.
+    a similar folder for your JSP templates used for this method. The other
+    template provided by the `AssetRenderer` interface, `preview.jsp`, is not
+    customized by the blogs asset renderer, so its default template is
+    implemented.
 
     You must configure a link to display the full content of the asset. You'll
-    do this later.
+    do this later. <!--Check -->
```

Comment:
@juliocamarero this was a personal note to myself. I was unsure whether the statement I made before it was correct. I'll remove it in the final draft. Let me know if the prior sentence is incorrect.